src/dmd/storage_class.d

```diff
@@ -0,0 +1,86 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
```

Comment:
I dream of a day where our module DDOC will actually be useful instead of always repeating the same sentence

Reply:
I see...